base/src/dispatcher/dispatcher.ts

```diff
@@ -118,7 +118,6 @@ export class Dispatcher implements ZLUX.Dispatcher {
              watchers[k].instanceRemoved(instance.applicationInstanceId);
            }
          }
-         return;
```

Comment:
return would prematurely leave the loop (so it becomes effectively an "if" statement instead of for loop).